packages/components/src/utils/events.ts

```diff
@@ -0,0 +1,50 @@
+type EventHandler< T extends Event > = ( event: T ) => void;
+
+/**
+ * Merges event handlers together.
+ *
+ * @template TEvent
+ * @param  handler
+ * @param  otherHandler
+ */
+function mergeEvent< TEvent extends Event >(
+	handler: EventHandler< TEvent >,
+	otherHandler: EventHandler< TEvent >
+): EventHandler< TEvent > {
+	return ( event: TEvent ) => {
+		if ( typeof handler === 'function' ) {
+			handler( event );
+		}
+		if ( typeof otherHandler === 'function' ) {
+			otherHandler( event );
+		}
+	};
+}
+
+/**
+ * Merges a set of event handlers together.
```

Comment:
Nit, but

```suggestion
 * Merges a pair of event handlers together.
```

Reply:
Also, is there ever going to be a need for this utility to merge more than 2 sets? i.e. should we consider changing the signature of the function to accept any number of objects to merge?

Reply:
No, you could just compose this call, right?

Reply:
Yes, that's true. Was mostly a curiosity in terms of API design — feel free to ignore my point :)